scripts/datastore_upgrade.py

```diff
@@ -409,6 +411,32 @@ def all_services_started(status_dict):
       return False
   return True
 
+def ensure_cassandra_nodes_match_replication(keyname):
+  # Get the database master IP.
+  with open(MASTERS_FILE_LOC) as masters_file:
```

Comment:
You can use `master_ip` var from the parent function. I'm going to open a PR that ensures that `master_ip` is the private IP for the db master since it's needed for cassandra.yaml.